services/gateway/apis/http-version3/swagger/definitions/transactions.json

```
@@ -368,5 +368,94 @@
                 "description": "Binary payload"
             }
         }
+    },
+    "dryTransactionWithEnvelope": {
+        "type": "object",
+        "properties": {
+            "data": {
+                "type": "object",
+                "properties": {
+                    "success": {
+                        "type": "boolean",
+                        "example": true
+                    },
+                    "events": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "data": {
+                                    "type": "object",
+                                    "properties": {
+                                        "senderAddress": {
+                                            "type": "string",
+                                            "example": "lskdwsyfmcko6mcd357446yatromr9vzgu7eb8y99",
+                                            "description": "The Lisk Address is the human-readable representation of a blockchain account.\nIt is 41 character long identifier that begins with `lsk`.\n"
+                                        },
+                                        "tokenID": {
+                                            "type": "string",
+                                            "example": "0000000000000000",
+                                            "description": "Token unique identifier."
+                                        },
+                                        "amount": {
+                                            "type": "string",
+                                            "example": "100003490",
+                                            "description": "Amount of Lisk (in Beddows) to be transferred in this transaction.\n"
+                                        },
+                                        "recipientAddress": {
+                                            "type": "string",
+                                            "example": "lskdwsyfmcko6mcd357446yatromr9vzgu7eb8y99",
+                                            "description": "The Lisk Address is the human-readable representation of a blockchain account.\nIt is 41 character long identifier that begins with `lsk`.\n"
```

Please update similar descriptions elsewhere.
```suggestion
                                            "description": "The Lisk Address is the human-readable representation of a blockchain account.\nIt is a 41 characters long identifier that begins with `lsk`.\n"
```